jenkins/image_building/build-image.sh

```diff
@@ -40,7 +40,7 @@ export DIB_DEV_USER_AUTHORIZED_KEYS="${current_dir}/authorized_keys"
 
 if [[ "${IMAGE_OS}" == "ubuntu" ]]; then
   export DIB_RELEASE=jammy
```

Comment:
```suggestion
  export DIB_RELEASE=noble
```

Shouldn't we have this?

Reply:
Ah missed that one!